app/views/hearings/show.html.haml

```diff
@@ -6,38 +6,19 @@
 
 .govuk-grid-row
   .govuk-grid-column-one-third
-    - unless FeatureFlag.enabled?(:hearing)
-      = render partial: 'details', locals: { hearing: @hearing, hearing_day: @hearing_day }
-    - else
-      = render partial: 'details_v2', locals: { hearing_summary: @prosecution_case.hearing_summaries.find { |each| each.id == @hearing_id },
+    = render partial: 'details_v2', locals: { hearing_summary: @prosecution_case.hearing_summaries.find { |each| each.id == @hearing_id },
```

Comment:
as before, it would be great to get rid off those `*_v2` views,
but I guess I may take time to fix/refactor relevant specs.

Up to you, we can do in a second step, I'll approve it.